libs/features/personalization/personalization.js

```diff
@@ -246,6 +248,26 @@ export function parseConfig(data) {
       variants[vn] = { commands: [] };
     });
 
+    const laterToDate = (laterExpression) => {
+      if (!laterExpression) return null;
+      const regex = /at (\d{1,2}):(\d{1,2}) on the (\d{1,2}) day of (\w+) in (\d{4})/;
```

Comment:
could you call it with more obvious name of what it means, not what type it is? `scheduleFormat` ?

Reply:
could we make that format shorter and more standard ? [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601)? YYYY-mm-ddThh:mm? This should save us a few bytes and be simpler

Reply:
>could you call it with more obvious name of what it means, not what type it is? scheduleFormat ?
sure, will rename it

>could we make that format shorter and more standard ? [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601)? YYYY-mm-ddThh:mm? This should save us a few bytes and be simpler
I can, but then we have 1 format in crontab.xlsx and another format in the manifest file. I chose to use same format that the Franklin team is using because then you can compare 2 dates in 2 places more quickly. 
not sure what is better.

Reply:
will the users that edit both be the same? one is fetched on client side, one not. I think this makes sense not aligning